lexers/troff.lua

```diff
@@ -0,0 +1,30 @@
+-- Copyright 2015-2017 David B. Lamkins <david@lamkins.net>. See LICENSE.
+-- man/roff LPeg lexer.
+
+local lexer = lexer
+local P, R, S = lpeg.P, lpeg.R, lpeg.S
+
+local lex = lexer.new(...)
+
+-- Registers and groff's structured programming.
+lex:add_rule('keywords', lex:tag(lexer.KEYWORD, (lexer.starts_line('.') * lexer.space^0 * (P('while') + P('break') + P('continue') + P('nr') + P('rr') + P('rnn') + P('aln') + P('\\}'))) + P('\\{'))
+
+-- Markup.
+lex:add_rule('escape_sequences', lex:tag(lexer.VARIABLE,
+	P('\\') * ((P('s') * S('+-')) + S('*fgmnYV') + P("")) *
+	(P('(') * 2 + P('[') * (lexer.any-P(']'))^0 * P(']') + 1))
+
+lex:add_rule('headings', lex:tag(lexer.NUMBER, P('.') * (S('STN') * P('H')) * (lexer.space + P("\n")) * lexer.nonnewline^0)
+lex:add_rule('man_alignment', lex:tag(lexer.KEYWORD, lexer.starts_line('.'), (P('br') + P('DS') + P('RS') + P('RE') + P('PD')) * (lexer.space + P("\n")))
```

Comment:
Your `lex:tag()` call has an additional parameter after `lexer.stars_line('.')` that will be ignored. Perhaps you meant to use a `*` instead of a `,`?